### ISSUE TITLE
Super slow

### ISSUE THREAD

**Issue by AtomicNess123** [jrblevin/deft#114]
I wonder if there is a problem with slowness. When calling M-x deft, it takes a long time to type.

**Comment by zougloub** [comment]
Yeah, I kind of wish there was an option to just ignore the reading the contents of all the deft files and showing the summary, but just rely on the file names.
For me deft-refresh takes a good 5 seconds and that's with the JIT and 4.4 GHz turbo boost.
Thankfully I only do that once at boot...
